.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: objective-c
+osx_image: xcode8.3
+xcode_sdk: iphonesimulator10.3
+xcode_workspace: workspace.xcworkspace
+xcode_scheme: test
+
+script:
+  - xcodebuild -workspace "workspace.xcworkspace" -scheme "test" clean build CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
+  - xcodebuild test -workspace "workspace.xcworkspace" -scheme "test" -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.3' CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO 
```

Comment:
Do a clean and build, then run all the unit tests in testTests. Not sure if UITests will ever be tested in CI but if so we will have to figure out why it was not working for me. I just removed UITests from the scheme.